src/pages/search/index.tsx

```diff
@@ -63,6 +62,7 @@ const Search: InferGetServerSidePropsType<typeof getServerSideProps> = ({ theme,
   const [showCSVDownloadPopup, setShowCSVDownloadPopup] = useState(false);
   const [showOptions, setShowOptions] = useState(false);
   const [drawerFamily, setDrawerFamily] = useState<boolean | number>(false);
+  const [searchChanged, setSearchChanged] = useState(false);
```

Comment:
Thought: In terms of variable naming, the word "dirty" usually describes a form or input whose value has updated from the starting value. "changed" also works, but just FYI.

Reply:
Good idea